codec-http/src/test/java/io/netty/handler/codec/http/HttpServerCodecTest.java

```diff
@@ -114,6 +114,60 @@ public void test100Continue() throws Exception {
         ch.finish();
     }
 
+    @Test
+    public void test100ContinueReject() {
+        HttpServerCodec decoder = new HttpServerCodec();
+        EmbeddedChannel channel = new EmbeddedChannel(decoder);
+        String oversized =
+                "PUT /file HTTP/1.1\r\n" +
+                        "Expect: 100-continue\r\n" +
+                        "Content-Length: 1048576000\r\n\r\n";
+
+        channel.writeInbound(Unpooled.copiedBuffer(oversized, CharsetUtil.US_ASCII));
+        assertThat(channel.readInbound(), is(instanceOf(HttpRequest.class)));
```

Comment:
you can actually check here for expected "413" code error response.

Reply:
no you can't as we do not send it...